src/Tools/SchemaValidator.php

```diff
@@ -53,7 +53,7 @@ class SchemaValidator
      */
     private const BUILTIN_TYPES_MAP = [
         AsciiStringType::class => 'string',
-        BigIntType::class => 'string',
+        BigIntType::class => 'string|int',
```

Comment:
We still support DBAL 3 though… maybe we should keep `string` when DBAL 3 is installed?

Reply:
Yeah, but how can the version of DBAL be detected here?

Reply:
You could test for the existence of `BigIntType::getName()` with a call to `method_exists()`. That method does not exist with DBAL 4.

Reply:
OK, but how can I do this inside this `const`?

Reply:
I don't think you can, you'd probably have to convert the const to a property.